client/src/api/user/index.ts

```diff
@@ -59,4 +71,10 @@ export const getUserPermissionsByID = async (
   return await get(url, failureMessage);
 };
 
-export { getUsers, getUser, updateUserClaimedPitches, updateUser };
+export {
+  getUsers,
+  getUser,
+  updateUserClaimedPitches,
+  updateUser,
+  updateOnboardingStatus,
```

Comment:
Could you export this from the `index.ts` file in `client/api/index.ts`

This makes it so that if someone needs this function from the frontend, they can just do:

`import { updateOnboardingStatus } from '../api';`

instead of

`import { updateOnboardingStatus} from '../api/users';`